WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -110,8 +113,6 @@ def __init__(self, *args, **kwargs):
     self.defaultSubmitPools = ''
     self.totalSubmittedPilots = 0
 
-    self.proxy = None
```

Comment:
This is still used in `submitPilots`

Reply:
I just changed it in `submitPilots`, it should be fine now.
I turned this object attribute into a variable because it was more specific to each queue (the attribute was always modified according to the queue processed). Is this okay for you?

Reply:
Fine for me!